<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: 2008-2011 Volker Lanz &lt;vl@fidra.de&gt;
SPDX-License-Identifier: GPL-3.0-or-later
 </author>
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">KDE Partition Manager</string>
  </property>
  <widget class="PartitionManagerWidget" name="m_PartitionManagerWidget"/>
  <widget class="QDockWidget" name="m_DockDevices">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string comment="@title:window">Devices</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ListDevices" name="m_ListDevices"/>
  </widget>
  <widget class="QDockWidget" name="m_DockOperations">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string comment="@title:window">Pending Operations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="ListOperations" name="m_ListOperations"/>
  </widget>
  <widget class="QDockWidget" name="m_DockInformation">
   <property name="windowTitle">
    <string comment="@title:window">Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="InfoPane" name="m_InfoPane"/>
  </widget>
  <widget class="QDockWidget" name="m_DockLog">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string comment="@title:window">Log Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="TreeLog" name="m_TreeLog"/>
  </widget>
  <widget class="QDockWidget" name="MessageWidgetDock">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="KMessageWidget" name="m_MessageWidget">
    <property name="messageType">
     <enum>KMessageWidget::Warning</enum>
    </property>
    <property name="position">
     <enum>KMessageWidget::Header</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>PartitionManagerWidget</class>
   <extends>QWidget</extends>
   <header>gui/partitionmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListDevices</class>
   <extends>QWidget</extends>
   <header>gui/listdevices.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListOperations</class>
   <extends>QWidget</extends>
   <header>gui/listoperations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeLog</class>
   <extends>QWidget</extends>
   <header>gui/treelog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoPane</class>
   <extends>QWidget</extends>
   <header>gui/infopane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
